stackslib/src/config/mod.rs

```diff
@@ -1236,115 +1236,121 @@ impl std::default::Default for Config {
 #[derive(Clone, Debug, Default, Deserialize, PartialEq)]
 pub struct BurnchainConfig {
     /// The underlying blockchain used for Proof-of-Transfer.
-    /// Currently, only `"bitcoin"` is supported.
-    ///
-    /// Default: `"bitcoin"`
+    /// ---
+    /// @default: `"bitcoin"`
+    /// @notes:
+    ///   - Currently, only `"bitcoin"` is supported.
     pub chain: String,
     /// The operational mode or network profile for the Stacks node.
     /// This setting determines network parameters (like chain ID, peer version),
     /// default configurations, genesis block definitions, and overall node behavior.
     ///
     /// Supported values:
-    /// ```text
-    /// | Mode            | Network Type |
-    /// |-----------------|--------------|
-    /// | "mainnet"       | mainnet      |
-    /// | "xenon"         | testnet      |
-    /// | "mocknet"       | regtest      |
-    /// | "helium"        | regtest      |
-    /// | "neon"          | regtest      |
-    /// | "argon"         | regtest      |
-    /// | "krypton"       | regtest      |
-    /// | "nakamoto-neon" | regtest      |
-    /// ```
-    ///
-    /// Default: `"mocknet"`
+    /// - `"mainnet"` → mainnet
+    /// - `"xenon"` → testnet
+    /// - `"mocknet"` → regtest
+    /// - `"helium"` → regtest
+    /// - `"neon"` → regtest
+    /// - `"argon"` → regtest
+    /// - `"krypton"` → regtest
+    /// - `"nakamoto-neon"` → regtest
```

Comment:
if the goal of this PR is to provide an easier text to parse/extract - is unicode chars the best choice to achieve that goal?

in this specific example, i can see that a markdown-ish table is not ideal for extracting the data, but would a more simple `:` or some other simple separator work better here?

happy to have to my mind changed if there is a tool/process  where the unicode arrow makes more sense.

Reply:
edit: i see in othe `@notes` you use a colon,i think that makes the most sense here
```suggestion
    /// - `"mainnet"`: mainnet
    /// - `"xenon"`: testnet
    /// - `"mocknet"`: regtest
    /// - `"helium"`: regtest
    /// - `"neon"`: regtest
    /// - `"argon"`: regtest
    /// - `"krypton"`: regtest
    /// - `"nakamoto-neon"`: regtest
```